docs/examples.rst

```diff
@@ -0,0 +1,38 @@
+Examples
+========
```

Comment:
Documentation should probably include (and maybe the examples changed) that visualisations can be included with the following statement:

    from webviz.page_elements import BarChart, LineChart, PieChart